src/views/DefaultContainer.vue

```diff
@@ -10,7 +10,9 @@
             <div v-if="displayEnv" class="p-3 bg-danger text-white text-uppercase">{{ environment }}</div>
             <b-navbar-nav class="ml-auto">
                 <b-nav-item v-if="hasHeaderFeedbackForm" class="px-2" v-b-modal.modal-give-feedback><i class="fa fa-comment"></i><div class="px-2 d-none d-sm-inline-block"> Give feedback</div></b-nav-item>
-                <b-nav-text v-if="profile !== null" class="px-3 d-none d-sm-inline-block">Welcome, {{ profile.nickname }}</b-nav-text>
+                <router-link to="Profile">
+                    <b-nav-text v-if="profile !== null" class="px-3 d-none d-sm-inline-block">Welcome, {{ profile.nickname }}</b-nav-text>
+                </router-link>
```

Comment:
b-nav-text ou b-nav-item directement avec un "to" en attribut ?

Reply:
Yes, si je fais ca le "welcome pseudo" devient gris comme le button logout. Et noir au hover. 
Boostrap-vue genere une balise <a> mais j'arrive pas lui faire passer du style pour rentre le text noir.